### ISSUE TITLE
Show Example with Sequential or Functional API

### ISSUE THREAD

**Issue by pGit1** [titu1994/Snapshot-Ensembles#1]
Can you show an example with Sequential or Functional API? The argparse syntax is very hard to read. This looks amazing! Thanks for the contribution.

Also in your code, it looks like you do the learning rate update after each epoch but the paper author(s) does it after each "training iteration". I am not sure what distinction they are making here.

**Comment by pGit1** [comment]
dumb request. Ignore.

**Comment by titu1994** [comment]
There's an example in the readme now.
